utils/utils.h

```diff
@@ -1,8 +1,9 @@
 #include <stdbool.h>
 #include <stdint.h>
+#include <X11/Xlib.h>
```

Comment:
Why is this in `utils.h`? I don't think its used here.

Reply:
Another mistake. I had originally planned to make it a utility, but found a better plan.